CMakeLists.txt

```diff
@@ -1027,7 +941,6 @@ foreach(__CLDNN_LinkerFlagName IN ITEMS "CMAKE_EXE_LINKER_FLAGS" "CMAKE_SHARED_L
       "${__CLDNN_LinkerFlagName}"
       ALL_PATTERN ""
       SET
-        DebugInfoProgramDatabase
```

Comment:
[MINOR] Similar to compiler options: **DebugInfoDisabled** (recommended)+ adding in this loop after (for Debug):
```
  intel_config_flag_apply_settings(
      LinkerOptions
      "${__CLDNN_LinkerFlagName}"
      PATTERN "^Debug"
      SET
        DebugInfoProgramDatabase
    )
```